movie.sh

```diff
@@ -1,12 +1,18 @@
 #!bin/bash
+
 function getrating()
 {
-curl "http://www.omdbapi.com/?t=$1&y=&plot=short&r=xml" > mov.txt
+#Fetch tuple from the Omdb API
+curl -s  "http://www.omdbapi.com/?t=$1&y=&plot=short&r=xml" > mov.txt
+#Getting the Line with the information
 grep -o imdbRating=".*" mov.txt > submov.txt
```

Comment:
If you change the above `curl` command on line 6 as I have suggested, what changes would you need to do here to make this work.